### PR TITLE
[df] Add GetColumnType function to distrRDF

### LINKED ISSUE #15039: [RDataFrame] Expose more local df operations for distributed RDF
### Explain what you would like to see improved and how.

There are many useful operations such as `GetColumnType` that is trivial to be included in the operation list of distributed RDF. I suggest to expose these operations to users.

The `GetColumnType` method is useful when someone tries to save columns based on the column type, e.g. I only want to save columns that are standard supported types for numpy arrays when I try to do `AsNumpy`. 

The very easy way to implement this would be to add it similar to how `GetColumnNames` is added [here](https://github.com/root-project/root/blob/a29e81cb1cd217ca2096a44d01fb273e085b4e8b/bindings/experimental/distrdf/python/DistRDF/HeadNode.py#L246).

A workaround for now would be to do
```
rdf._headnode._localdf.GetColumnType
```

### ROOT version

6.30/04

### Installation method

LCG (Swan)

### Operating system

Linux

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by martamaja10**
Simple addition of the GetColumnType function to distributed RDF. 

This PR fixes #15039

### CODE REVIEW

**Review by vepadulano — CHANGES_REQUESTED** [review]
Thank you this is a nice addition! Can you also provide a simple test for it?

File: bindings/experimental/distrdf/test/test_proxy.py [hunk 1]
```diff
@@ -272,3 +272,11 @@ def test_getcolumnnames_after_define(self):
...
 
         self.assertSequenceEqual(cols_after, ["x", "y", "z"])
+        
+    def test_get_column_type(self):
+        
+        hn = create_dummy_headnode(self.test_treename, self.test_filename)
+        proxy = Proxy.NodeProxy(hn)
+        column_type = proxy.GetColumnType("myColumn")
+        self.assertSequenceEqual(column_type, "Float_t")
+
```
Comment (vepadulano) [inline bindings/experimental/distrdf/test/test_proxy.py:282]: We should add another test with a column created with `Define`

**Review by vepadulano — CHANGES_REQUESTED** [review]
Nice, thanks! But see comment

File: bindings/experimental/distrdf/test/test_proxy.py (hunk 1 above)
Comment (martamaja10) [reply bindings/experimental/distrdf/test/test_proxy.py:307]: yes, added now.

**Review by vepadulano — APPROVED** [review]
Thank you!

**Pull request merged**